src/commands/projects.ts

```diff
@@ -169,26 +169,35 @@ const list = async (props: CommonProps & { orgId?: string }) => {
     return result;
   };
 
-  const ownedProjects = getList(props.apiClient.listProjects);
+  const ownedProjects = await getList(props.apiClient.listProjects);
   const sharedProjects = props.orgId
-    ? undefined
-    : getList(props.apiClient.listSharedProjects);
+    ? []
+    : await getList(props.apiClient.listSharedProjects);
 
   const out = writer(props);
 
-  out.write(await ownedProjects, {
-    fields: PROJECT_FIELDS,
-    title: 'Projects',
-  });
+  if (ownedProjects.length) {
+    out.write(ownedProjects, {
+      fields: PROJECT_FIELDS,
+      title: 'Projects',
+    });
+    out.end();
```

Comment:
I'm not 100% sure about moving `.out()` here, but seems that is the only way to ensure we output the table first and then the text. Otherwise, we'd need to have `flush()` or something similar copying `end()` functionality